src/components/LocationCard/LocationCard.js

```diff
@@ -13,6 +13,7 @@ export const LocationCard = (props) => {
         As,
         locationId,
         locationTitle,
+        hasRadiusSlider,
         distanceRadius,
         sliderLabel,
```

Comment:
We have a contract between input and label for all our inputs.
When we click on the label we have to make input active. For our case click on the label should activate the modal window. 
Actually, I don't like the idea that we use not` input` elements for `LocationSelector`. 
For my point is better to split up logic `input` (Reach) + `LocationSelector` (OneUI).